### PR TITLE
IPv6 highlight issue

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29690

## Description
Fixed some highlight (yellow color) issues.
Fixed some of the issues - https://github.com/demisto/etc/issues/29690

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ronykoz** [comment]
update the fixes link

**Review by anara123 — CHANGES_REQUESTED** [review]
image
The regex still not good enough to catch ipv6 properly. It still catches false positives

**Comment by ChanochShayner** [comment]
@anara123 
So is there any other change to make in regex?
I tried for a long time to fix it and failed.

**Comment by ronykoz** [comment]
@ChanochShayner yes, we need to fix it so that only the right regexes will pan out.

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+args:
+- default: true
+  description: The ipv6 to vriefy.
```
Comment (ronykoz) [inline Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.yml:3]:
```suggestion
  description: The IPv6 to vriefy.
```

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    for item in the_input:
+        if is_valid_ipv6_address(item):
+            entries_list.append(item)
+        else:
+            continue
+
+    if entries_list:
+        demisto.results(entries_list)
+    else:
+        demisto.results('')
```
Comment (ronykoz) [inline Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py:29]: i think it should be an empty list

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+from CommonServerPython import *
+import ipaddress
+
+
+def is_valid_ipv6_address(address):
+    try:
+        ipaddress.IPv6Address(address)
+        return address
+    except ValueError:
+        return
```
Comment (ronykoz) [inline Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py:11]: use True False

**Review by ronykoz — CHANGES_REQUESTED** [review]
Let's go over the TPB together

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.yml (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.yml]: done

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py (hunk 2 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py]: done

File: Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py (hunk 3 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/VerifyIPv6Indicator/VerifyIPv6Indicator.py]: done

**Review by ronykoz — APPROVED** [review]

**Comment by kirbles19** [comment]
@ChanochShayner @ronykoz Done.

**Review by anara123 — APPROVED** [review]

**Pull request merged**